src/foam/nanos/logger/LogMessageController.js

```diff
@@ -0,0 +1,40 @@
+foam.CLASS({
+  package: 'foam.nanos.logger',
+  name: 'LogMessageController',
+  extends: 'foam.comics.DAOController',
+
+  documentation: 'A custom DAOController to work with log messages.',
+
+  requires: ['foam.nanos.logger.LogMessage'],
+
+  imports: ['logMessageDAO'],
+
+  properties: [
+    {
+      class: 'foam.dao.DAOProperty',
+      name: 'data',
+      factory: function() {
+        return this.logMessageDAO.orderBy(this.LogMessage.CREATED);
+      }
+    },
+    {
+      name: 'summaryView',
+      factory: function() {
+        return this.FixedWidthScrollTableView;
+      }
+    }
+  ],
+
+  classes: [
```

Comment:
Is this new class needed? Couldn't you just scope the CSS in the LogMessageController?

Reply:
I tried that but it doesn't work because `LogMessageController` extends `DAOController`, not `DAOControllerView`. And since it's not a view, it doesn't get CSS.

Reply:
I see. Another reason to change comics to get rid of the view/model distinction, which doesn't add any value and just causes extra work and complexity.

Reply:
I guess since the fixed column width PR was merged we don't need this anymore anyway, so I'll remove it.